rosidl_generator_c/resource/msg__functions.c.em

```diff
@@ -81,7 +82,10 @@ for field in spec.fields:
         # non-array field
         if field.type.is_primitive_type():
             if field.type.type == 'string':
-                lines.append('rosidl_generator_c__String__init(&msg->%s);' % field.name)
+                lines.append('if (!rosidl_generator_c__String__init(&msg->%s)) %s' % (field.name, '{'))
+                lines.append('  return false;')
```

Comment:
Isn't it necessary to execute the `cleanup_lines` in case of a `return` to clean up already initialized parts?

Same below.